flow/environment.py

```diff
@@ -105,6 +111,7 @@ class ComputeEnvironment(metaclass=ComputeEnvironmentType):
     hostname_pattern = None
     submit_flags = None
     template = 'base_script.sh'
+    mpi_cmd_string = 'mpiexec'
```

Comment:
Why is this not called `mpi_prefix` like everywhere else?

Reply:
`mpi_cmd_string` is meant for just the system specific mpi syntax like `mpiexec` or `ibrun` or `jsrun` etc. And `mpi_prefix` remain to mean the whole mpi prefix such as `ibrun -n -4`

Reply:
Got it, can we rename this to `mpi_cmd` ?

Reply:
that was what i did before but I later found out there is a function called mpi_cmd already for some legacy code in the environment class. Would that be a issue reuse the name? If you think this is ok i can change them to back mpi_cmd.

Reply:
@csadorf the `legacy_templating.py` file indicates that we intended to remove the legacy templating functionality as of version 0.8... which we are currently on. This PR was never intended to be part of the current release of flow, but rather part of the next version 0.9 which is hopefully where groups fit in as well. Should we consider removing the legacy templating prior to merging this PR?

Reply:
Yes.